src/Providers/ReviewsIo/Request/Service/EmailInviteRequest.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
Trustpilot InviteRequest file is called `InviteRequest.php` but ReviewsIo InviteRequest file is called `EmailInviteRequest.php`, I would rename to `InviteRequest.php` to have both be the same